randr.go

```diff
@@ -144,6 +170,41 @@ func (m Modes) String() string {
 	return strings.Join(str, " ")
 }
 
+// Generates the monitor id from the edid
+func GenerateMonitorId(edid string) (string, error) {
+	var errEdidCorrupted = errors.New("Edid corrupted: " + edid)
+	if edid[:16] != "00ffffffffffff00" {
```

Comment:
A length check would be great here, that the string has the right length. Otherwise the slicing here will panic.